samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties

```diff
@@ -178,6 +178,8 @@ samigo_formula_error_5=5\: Parte inesperada
 samigo_formula_error_6=6\: Final inesperado de la expresi\u00f3n 
 samigo_formula_error_7=7\: Valor esperado
 samigo_formula_error_9=9\: Variable no definida en las instrucciones
+samigo_formula_error_10=10: Variable no definida en el comentario a la respuesta correcta
+samigo_formula_error_11=11: Variable no definida en el comentario a la respuesta incorrecta
```

Comment:
Could you add these strings in the _ca and _eu files?

Reply:
Done!